p2p/session.go

```diff
@@ -168,6 +168,7 @@ func (s *session[H]) doRequest(
 			logFn = log.Debugw
 			fallthrough
 		case errEmptyResponse:
+			logFn = log.Debugw
```

Comment:
May be let's rework switch to if else ?

Reply:
sure i'll try it out and you can see if you like it better

Reply:
just delete L168

Reply:
omg i'm blind

Reply:
still a switch but now simpler. I think an if would be uglier here, but can still change it if you guys prefer